examples/logical_enclosure.rb

```diff
@@ -51,6 +51,7 @@
   client my_client
   script '# My script commands here'
   action :set_script
+  only_if { client[:api_variant] == 'C7000' }
```

Comment:
Add the same kind of validation to few examples which are mentioned only for synergy variant. I have seen create LE and delete LE mentioned only for synergy. Please add if condition for those examples also.